test/api_integration/apis/kql_telemetry/kql_telemetry.ts

```diff
@@ -65,7 +70,7 @@ export default function ({ getService }: FtrProviderContext) {
         })
         .then((response) => {
           const kqlTelemetryDoc = get(response, 'hits.hits[0]._source.kql-telemetry');
-          expect(kqlTelemetryDoc.optOutCount).to.be(1);
+          expect(kqlTelemetryDoc.optOutCount).to.be.greaterThan(0);
```

Comment:
same as above, what is the reason for this change?

Reply:
reverted.